src/theme/CodeBlock/Content/String.js

```diff
@@ -49,7 +49,7 @@ export default function CodeBlockString({
     >
       {title && <div className={styles.codeBlockTitle}>{title}</div>}
       <div className={styles.codeBlockContent}>
-        <Highlight {...defaultProps} theme={prismTheme} code={code} language={language ?? "text"}>
```

Comment:
I investigated this one.  `defaultProps` is not being exported anymore. The old defaultProps.js file just has the following content:

```
import Prism from "prism-react-renderer/prism";
import theme from "prism-react-renderer/themes/duotoneDark"; // Adjust the theme import based on availability

<Highlight Prism={Prism} theme={theme} code={code} language={language ?? "text"}>
    {/* rest of your component code */}
</Highlight>

```

I haven't found this to really do anything? so I simply removed it.